.strict-typing

```diff
@@ -146,6 +146,7 @@ homeassistant.components.image_processing.*
 homeassistant.components.input_button.*
 homeassistant.components.input_select.*
 homeassistant.components.integration.*
+homeassistant.components.islamic_prayer_times.*
```

Comment:
Maybe strict typing should be a separate PR. It feels like this PR is getting harder to review.

Reply:
Sure I will revert it.